open-metadata-implementation/governance-servers/open-lineage-services/open-lineage-services-server/src/main/java/org/odpi/openmetadata/governanceservers/openlineage/admin/OpenLineageServerOperationalServices.java

```diff
@@ -189,21 +187,9 @@ private Connection getAssetLineageOutTopicConnection(OpenLineageServerConfig ope
         }
 
         VirtualConnection assetLineageConnection = (VirtualConnection) restResult.getConnection();
-        Connection assetLineageTopicConnectionOverride = openLineageServerConfig.getAssetLineageTopicConnectionOverride();
-        if (assetLineageConnection != null && assetLineageTopicConnectionOverride != null) {
-            Map<String, Object> configurationProperties = assetLineageTopicConnectionOverride.getConfigurationProperties();
-
-            List<EmbeddedConnection> embeddedConnections = assetLineageConnection.getEmbeddedConnections();
-            if (embeddedConnections != null) {
-                for (EmbeddedConnection embeddedConnection : embeddedConnections) {
-                    Connection connection = embeddedConnection.getEmbeddedConnection();
-                    if (connection != null && KAFKA_OPEN_METADATA_TOPIC_PROVIDER.equalsIgnoreCase(connection.getConnectorType().getConnectorProviderClassName())) {
-                        connection.setConfigurationProperties(configurationProperties);
-                        embeddedConnection.setEmbeddedConnection(connection);
-                    }
-                }
-            }
-            assetLineageConnection.setConfigurationProperties(configurationProperties);
+        Connection inTopicConnection = openLineageServerConfig.getInTopicConnection();
```

Comment:
This check should be performed before the lookup with the omas client takes place. If locally configured then this method returns and does not perform remote call thus server should create the connection  and start independently.

Reply:
Referring to the check at line 191.